src/components/Button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
it will be good if you create a folder called like 'dumyComponents' and put this component inside it in components folder

Reply:
This is not a dummy file as it is a well styled button but I can also do that and move it to a dummy components section.